standards/readme-guidelines.md

```diff
@@ -67,6 +67,3 @@ Supports printing commands for the TSP100 Thermal Receipt Printer.
   - Copy the release notes into the release description textbox
   - Attach the installer zip file in the upload area
   - Click `Publish Release`
-
-## Wiki
```

Comment:
So this section was supposed to represent part of the Readme example, showing that we should leverage Wikis for extended details and additional readme content. Instead of deleting this wholesale, can we think of a way to improve how that messaging comes across?

Reply:
Put something in the wiki for this repo and link it there?

Reply:
What was in the `Magtek Card Reader Middleware` Project? Or should we just make something up that looks like it was an extension of the project?

Reply:
An intro, detailed installation instructions, API documentation, and resources. I think we could create something generic and similar in the wiki for this repo. This repo is public, but the one being linked to was not.

Reply:
Can I get the original docs for this project? 

Ill go ahead and create a wiki page with the details and link it here.

Reply:
Fixed via: e046c55cc080e822e257cf3187b57e763ea277e1

Reply:
Looks good to me. We can fill out the wiki with something generic and useful.